src/j1client.js

```diff
@@ -103,7 +114,30 @@ class JupiterOneClient {
       if (res.errors) {
         throw new Error(`JupiterOne returned error(s) for query: '${j1ql}'`);
       }
-      const { data } = res.data.queryV1;
+
+      const deferredUrl = res.data.queryV1.url;
+      let status = JobStatus.IN_PROGRESS;
+      let statusFile;
+      var sleep = function(ms) {
+        return new Promise(function(resolve) {
+          return setTimeout(resolve, ms);
+        });
+      };
+      do {
```

Comment:
Please add logic for max number of attempts

Reply:
The max number of attempts can be really high but we probably want it to eventually fail. Maybe try for up to 5 minutes? You could keep track of the start time and if the elapsed time is over 5 minutes then stop trying.

Reply:
Added logic for 5 minutes of retries.